public/theme/boltcms-2021/listing.twig

```diff
@@ -0,0 +1,32 @@
+{# @var version \App\VersionControl #}
```

Comment:
Rename to `news.twig`, so that it works for the listing of news, but not other CTs.